tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -287,35 +300,28 @@ def test_fit(self, toy_X: pd.DataFrame, toy_y: pd.Series) -> None:
         n_channel: int = len(mmm.channel_columns)
         n_control: int = len(mmm.control_columns)
         fourier_terms: int = 2 * mmm.yearly_seasonality
-        mmm.fit(
-            X=toy_X,
-            y=toy_y,
-            target_accept=0.81,
-            draws=draws,
-            chains=chains,
-            random_seed=rng,
-        )
+        mmm = mock_fit(mmm, toy_X, toy_y.to_numpy())
         idata: az.InferenceData = mmm.fit_result
         assert (
             az.extract(data=idata, var_names=["intercept"], combined=True)
             .to_numpy()
             .size
-            == draws * chains
+            == 500
```

Comment:
Can we add a comment on where this 500 is coming from? Can we keep the `chains * draws` to make it more transparent?